src/js/components/Button/__tests__/__snapshots__/Button-kind-test.js.snap

```diff
@@ -872,7 +872,7 @@ exports[`Button kind badge should render custom element 1`] = `
           viewBox="0 0 24 24"
         >
           <path
-            d="M12 22V2M2 12h20"
+            d="M12,22 L12,2 M2,12 L22,12"
```

Comment:
Why is there a snapshot change in this SVG?

Reply:
@ericsoderberghp I'm not sure, it came when I ran `yarn test -update`

Reply:
@jeferson-sb I suspect you need to make sure you are referencing the latest Grommet dependencies, specifically grommet-icons. Can you make sure you have run `yarn cache clean && yarn` recently? Then run the `yarn test --update`.

Suspicion is due to how the paths are changing and are no longer SVG optimized.

Reply:
@halocline cool, that works! thank you